services/galley/src/Galley/Cassandra/TeamFeatures.hs

```diff
@@ -140,13 +140,27 @@ setSelfDeletingMessagesStatus tid status = do
           <> ", self_deleting_messages_ttl) "
           <> "values (?, ?, ?)"
 
+setSelfDeletingMessagesPaymentStatus ::
```

Comment:
```suggestion
-- FUTUREWORK: this function can be made polymorphic in the `TeamFeatureName` in the spirit
-- of `setFeatureStatusNoConfig` (defined futher up in this module).
setSelfDeletingMessagesPaymentStatus ::
```

It's ok not to do that now, but when we get to making the next feature (un-)lockable we don't want to duplicate this function, but re-use it.

Come to think of it: we should make this polymorphic now, then the implementation of the handler function that's calling this will also become easier: `setPaymentStatus` won't need the second argument, but can always call this polymorphic function without worrying about the concrete `TeamFeatureName`.

Reply:
done